src/DssSpell.sol

```diff
@@ -18,24 +18,31 @@ pragma solidity 0.8.16;
 
 import "dss-exec-lib/DssExec.sol";
 import "dss-exec-lib/DssAction.sol";
-
 import { GemAbstract } from "dss-interfaces/ERC/GemAbstract.sol";
-import { VatAbstract } from "dss-interfaces/dss/VatAbstract.sol";
 
-interface ProxyLike {
-    function exec(address target, bytes calldata args) external payable returns (bytes memory out);
+interface RwaLiquidationLike {
+    function ilks(bytes32) external view returns (string memory, address, uint48, uint48);
+    function init(bytes32, uint256, string calldata, uint48) external;
+}
+
+interface InputConduitJarLike {
+    function push(uint) external;
+}
+
+interface JarLike {
+    function void() external;
 }
 
 contract DssSpellAction is DssAction {
     // Provides a descriptive tag for bot consumption
     // This should be modified weekly to provide a summary of the actions
-    // Hash: cast keccak -- "$(wget 'https://raw.githubusercontent.com/makerdao/community/052330ed0eb8680dd9e4efaf92a4c056e46a4284/governance/votes/Executive%20vote%20-%20January%2024%2C%202024.md' -q -O - 2>/dev/null)"
+    // Hash: cast keccak -- "$(wget 'https://raw.githubusercontent.com/makerdao/community/4fa152956a992fe19ce1e8cce89920fcf896b304/governance/votes/Executive%20vote%20-%20February%2007%2C%202024.md' -q -O - 2>/dev/null)"
     string public constant override description =
-        "2024-01-24 MakerDAO Executive Spell | Hash: 0x17cf7409b1ad2b06b8d3fdbe943ffc65ddc933b57674880c85b2a3d8168e1bf4";
+        "2024-02-07 MakerDAO Executive Spell | Hash: 0xeafae0d525faaab32a2e67aeef2aff3cd59b80b2c466553986fe72917f4b14b7";
```

Comment:
Note: the url and the hash would need to be changed due to the minor issue in the exec doc. More info in [the discord](https://discord.com/channels/893112320329396265/1201914888709210192/1204728271262982144)

Reply:
A new exec doc was merged, note the change in the file name: https://github.com/makerdao/community/blob/master/governance/votes/Executive%20vote%20-%20February%207%2C%202024.md

Reply:
Fixed in cfd0855